### PR TITLE
Assignment 3 warmup

### PR DESCRIPTION
**Pull request by gpsingh12**

### CODE REVIEW

File: task_12.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Docstring."""
+FLOATVAL = 0.1
+INTVAL = 1
+import decimal
+DECVAL = decimal.Decimal("0.1")
+import fractions
```
Comment (cheuschober) [inline task_12.py]: It's almost universally considered good form to put your imports at the top of the and not in the middle.

File: task_12.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+FRACVAL = fractions.Fraction("1/10")
```
Comment (cheuschober) [inline task_12.py]: While this is one way of constructing a fraction, another way would be as `fractions.Fraction(1, 10)`

Comment (gpsingh12) [inline task_12.py]: Prof, 
I tried that too but it was printing Fraction(1, 10) as output.

Comment (cheuschober) [inline task_12.py]: @gpsingh12, We don't care about how it `print` -s -- it's a very important point to understand that printing is for humans and does not always exactly represent how the *data* is *stored*.

Comment (gpsingh12) [inline task_12.py]: Sounds good.
Thanks.

**Pull request merged**